mage_integrations/mage_integrations/sources/pipedrive/templates/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "api_token": "",
+    "start_date": "2020-01-01T00:00:00Z"
+}
```

Comment:
nit: add an empty line to the end of the file

Reply:
Fixed on commit 5032fbfa4c5701035425bc14e47fc2a6d4897da5